packages/react/theme/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "private": true,
```

Comment:
Using this for name-spaced import, `'@aws-amplify/ui-react/theme'`. This is because RSC look at the entire import dependency tree whether or not you use all that code, so if we want anything to work with RSC it can't come from the main export.